### PR TITLE
BRIDGE-2024: Add a request error metric

### PR DESCRIPTION
**Pull request by zaro0508**
Setup email notification when there are request errors.

### CODE REVIEW

File: cf_templates/eb_app.yml [hunk 1]
```diff
@@ -466,6 +466,77 @@ Resources:
...
+      EvaluationPeriods: 1
+      MetricName: !Join
+        - '-'
+        - - !Ref 'AWS::StackName'
+          - RequestErrorCount
+      Namespace: LogMetrics/Errors
+      Period: 43200
+      Statistic: Sum
+      Threshold: 10
+      TreatMissingData: notBreaching
```
Comment (zaro0508) [inline cf_templates/eb_app.yml]: Need to manually set DefaultValue parameter, https://sagebionetworks.jira.com/browse/BRIDGE-1986

File: cf_templates/eb_app.yml [hunk 2]
```diff
@@ -466,6 +466,77 @@ Resources:
...
+      EvaluationPeriods: 1
+      MetricName: !Join
+        - '-'
+        - - !Ref 'AWS::StackName'
+          - RequestWarningCount
+      Namespace: LogMetrics/Warnings
+      Period: 43200
+      Statistic: Sum
+      Threshold: 100
+      TreatMissingData: notBreaching
```
Comment (zaro0508) [inline cf_templates/eb_app.yml]: same here

File: cf_templates/eb_app.yml [hunk 3]
```diff
@@ -466,6 +466,77 @@ Resources:
...
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      EvaluationPeriods: 1
+      MetricName: !Join
+        - '-'
+        - - !Ref 'AWS::StackName'
+          - RequestErrorCount
+      Namespace: LogMetrics/Errors
+      Period: 43200
+      Statistic: Sum
+      Threshold: 10
```
Comment (DwayneJengSage) [inline cf_templates/eb_app.yml]: Similarly, this should be 10 errors in 1 hour.

File: cf_templates/eb_app.yml [hunk 4]
```diff
@@ -466,6 +466,77 @@ Resources:
...
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      EvaluationPeriods: 1
+      MetricName: !Join
+        - '-'
+        - - !Ref 'AWS::StackName'
+          - RequestWarningCount
+      Namespace: LogMetrics/Warnings
+      Period: 43200
+      Statistic: Sum
+      Threshold: 100
```
Comment (DwayneJengSage) [inline cf_templates/eb_app.yml]: And this should be 100 warnings in 1 hour.

**Review by DwayneJengSage — CHANGES_REQUESTED** [review]

**Review by DwayneJengSage — APPROVED** [review]

**Pull request merged**